### ISSUE TITLE
Error when trying to load .tbm's after restarting game. 

### ISSUE THREAD

**Issue by Mulazo77** [0ceal0t/Dalamud-VFXEditor#29]
Anyone get this error before? Any solutions? Happens when I try to load in my own saved ones to tweak them etc. 

--------------------------------
19:53:13.572 | INF [VFXEditor] Loading XivTmb
--------------------------------
19:53:18.546 | ERR ["VFXEditor"] UiBuilder OnBuildUi caught exception
System.OverflowException: Arithmetic operation resulted in an overflow.
   at Lumina.Data.SqPackStream.ReadFile[T](Int64 offset)
   at Lumina.Data.SqPack.ReadFile[T](Int64 offset)
   at Lumina.Data.Category.GetFile[T](ParsedFilePath path)
   at Lumina.Data.Repository.GetFile[T](Byte cat, ParsedFilePath path)
   at Dalamud.Data.DataManager.GetFile[T](String path) in C:\goatsoft\companysecrets\dalamud\Data\DataManager.cs:line 121
   at VFXEditor.Tmb.TmbDocument.LoadGame(String gamePath)
   at VFXEditor.FileManager.FileManagerDocument`2.SetSource(SelectResult result)
   at VFXEditor.Tmb.TmbManager.<>c.<Setup>b__8_0(SelectResult result)
   at VFXSelect.VFX.TmbActionSelect.DrawSelected(XivTmb loadedItem)
   at VFXSelect.SelectTab`2.Draw()
   at VFXSelect.SelectDialog.DrawGame()
   at VFXSelect.SelectDialog.Draw()
   at VFXEditor.Tmb.TmbManager.DrawBody()
   at VFXEditor.Dialogs.GenericDialog.Draw()
   at VFXEditor.Plugin.Draw()
   at Dalamud.Interface.UiBuilder.OnDraw() in C:\goatsoft\companysecrets\dalamud\Interface\UiBuilder.cs:line 287
{ } [ Send ]

**Comment by 0ceal0t** [comment]
looks like an issue with the TexTools mod export. I'll check for myself and see if I can reproduce it

**Comment by Mulazo77** [comment]
Figured it out! It fails to load the .tmb if you are actively USING the mod you made. i.e. I changed Heat Blast and exported as a .ttmp. Went back into game, the changes worked perfectly but I wanted to fix one small thing, tried to load .tmb for it and it crashed. When I disable the mod in textools and go back in, loading the .tmb works fine and I can continue working on it. Silly mistake on my part.
